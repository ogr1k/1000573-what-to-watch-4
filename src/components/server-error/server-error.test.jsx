import React from "react";
import renderer from "react-test-renderer";
import ServerError from "./server-error.jsx";


it(`Render ShowMoreButton`, () => {
  const tree = renderer
    .create(<ServerError />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
