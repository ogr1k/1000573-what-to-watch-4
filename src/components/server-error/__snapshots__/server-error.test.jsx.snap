// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render ShowMoreButton 1`] = `
<div
  id="servererror"
>
  <div
    className="servererror"
  >
    <div
      className="servererror-message"
    >
      <h1>
        Oops!
      </h1>
      <h2>
        Server is unavailable
      </h2>
    </div>
    <a
      href="#"
      onClick={[Function]}
    >
      Refresh page
    </a>
  </div>
</div>
`;
